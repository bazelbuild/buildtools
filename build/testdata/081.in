manifest = [
   # buildifier: table sort 1
   ("ab", "bc", "cd"),
   ("a",  "bd",  "ff"),
   ("a",  "b",  "cc"),
]
