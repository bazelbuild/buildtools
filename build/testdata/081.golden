manifest = [
    # buildifier: table sort 1
    ("a",     "bd",    "ff"),
    ("a",     "b",     "cc"),
    ("ab",    "bc",    "cd"),
]
